_validate/createJson.py

```diff
@@ -35,6 +35,7 @@ def generateJsonFile(
 		url: str,
 		licenseName: str,
 		licenseUrl: typing.Optional[str],
+		legacy: typing.Optional[bool],
```

Comment:
I don't think legacy should be included in create JSON - I don't think we should allow or encourage more legacy submissions to be created.